cocos2d/core/platform/CCSys.js

```diff
@@ -708,6 +710,15 @@ else {
                         _supportWebGL = true;
                     }
                     break;
+                case sys.BROWSER_TYPE_CHROME:
+                    // Chrome on android supports WebGL from v. 30
+                    var browserVer = parseFloat(sys.browserVersion);
```

Comment:
Can you take this out of switch ? Now two case is using the same variable, to avoid multiple declaration, it's better to take it out